sdk/core/Azure.Core.TestFramework/README.md

```diff
@@ -265,20 +265,29 @@ For example:
     }
 ```
 
-Another sanitizer property that is available for sanitizing Json payloads is the `JsonPathSanitizers`. This property contains a list of [Json Path](https://www.newtonsoft.com/json/help/html/QueryJsonSelectToken.htm) format strings that will be validated against the body. If a match exists, the value will be sanitized.
+Another sanitizer feature that is available for sanitizing Json payloads is the `AddJsonPathSanitizer`.
+This method allows adding a [Json Path](https://www.newtonsoft.com/json/help/html/QueryJsonSelectToken.htm) format strings that will be validated against the body. If a match exists, the value will be sanitized.
 
 ```c#
     public class FormRecognizerRecordedTestSanitizer : RecordedTestSanitizer
     {
         public FormRecognizerRecordedTestSanitizer()
             : base()
         {
-            JsonPathSanitizers.Add("$..accessToken");
-            JsonPathSanitizers.Add("$..source");
+            AddJsonPathSanitizer("$..accessToken");
+            AddJsonPathSanitizer("$..source");
         }
     }
 ```
 
+Sometimes it's useful to be able to have a custom replacement values for JsonPath-based sanitization (connection strings, JWT tokens).
+To enable this the `AddJsonPathSanitizer` provides an additional callback that would be called for every match.
```

Comment:
We don't actually need the callback in the place this is used - we just need the ability to specify some static sanitization string. Not saying it may not be needed eventually, but the needed feature could be satisfied with a simpler API.

Reply:
FYI @AlexanderSher  in case CredScan starts flagging this things, in which case something standard will have to be added